### PR TITLE
Added vehicle calibration info

### PR DESCRIPTION
**Pull request by bhavya01**
- Added calibration info for all the sensors. 
- The camera sensors for nuScenes does not have distortion coefficients. Output zeros for those.
- nuScenes using one lidar compared to two for argoverse. Output the same transformation for both up and down lidar in argoverse.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -10,14 +10,50 @@
...
     'CAM_FRONT_LEFT': 'ring_front_left',
     'CAM_FRONT_RIGHT': 'ring_front_right',
     'CAM_BACK_LEFT': 'ring_side_left',
     'CAM_BACK_RIGHT': 'ring_side_right',
 }
 
+def is_camera_sensor(sensor):
+    return sensor[0:3] == "CAM"
+
+def get_calibration_info(nusc, scene):
```
Comment (johnwlambert) [inline main.py]: thanks for this PR, Bhavya. Do you mind adding type hints to the input and return arguments to each function?

Here are a few pointers:
https://docs.python.org/3/library/typing.html
https://www.python.org/dev/peps/pep-0484/

File: main.py [hunk 2]
```diff
@@ -10,14 +10,50 @@
 NUSCENES_ROOT = 'nuscenes'
 
 SENSOR_NAMES = {
-    'LIDAR_TOP': 'lidar'
+    'LIDAR_TOP': 'lidar',
```
Comment (johnwlambert) [inline main.py:31]: do you mind adding a small inline comment above `SENSOR_NAMES` explaining the mapping (from nuscenes to argoverse, for example, nuscenes sensor on left, corresponding argoverse sensor on right)

File: main.py [hunk 3]
```diff
@@ -10,14 +10,50 @@
 NUSCENES_ROOT = 'nuscenes'
```
Comment (johnwlambert) [inline main.py]: can we make `NUSCENES_ROOT` and `OUTPUT_ROOT` arguments to argparse`? Fine to keep the defaults to the values you have listed here in the `argparse` commands

Comment (johnwlambert) [reply main.py]: do you also mind using the PEP8 import layout? https://www.python.org/dev/peps/pep-0008/#imports
```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
```

File: main.py [hunk 4]
```diff
@@ -10,14 +10,50 @@
...
-    'LIDAR_TOP': 'lidar'
+    'LIDAR_TOP': 'lidar',
     'CAM_FRONT': 'ring_front_center',
     'CAM_FRONT_LEFT': 'ring_front_left',
     'CAM_FRONT_RIGHT': 'ring_front_right',
     'CAM_BACK_LEFT': 'ring_side_left',
     'CAM_BACK_RIGHT': 'ring_side_right',
 }
 
+def is_camera_sensor(sensor):
```
Comment (johnwlambert) [inline main.py]: i might call the input arg `sensor_name` just to make it a bit clearer

**Review by johnwlambert — CHANGES_REQUESTED** [review]

**Comment by johnwlambert** [comment]
Hi Bhavya, do you mind adding a docstring at the start of the file just mentioning what sample and sweep mean, and a bit of other high-level info:
```python
"""
Converts the nuScenes dataset into the Argoverse format.

Samples are... at XXX hz and sweeps are... ...which is annotated...
LiDAR is provided in XXX coordinate system, annotations are provided in XXX coordinate systema, and images are undistorted?
"""
```

File: main.py (hunk 1 above)
Comment (bhavya01) [reply main.py]: Added in the latest commit

File: main.py (hunk 3 above)
Comment (bhavya01) [reply main.py]: Added in the latest commit

**Review by johnwlambert — APPROVED** [review]

**Pull request merged**